packages/TelegramClient-Tests.package/TCTAcceptanceTests.class/instance/testMessageAuthors.st

```diff
@@ -4,14 +4,14 @@ testMessageAuthors
 	| message user chat_id notification |
 	self receivedMessage: nil.
 	chat_id := ((self core chats) first) id.
-	message := self class mockMessageFrom: chat_id.
+	message := TCTMocks mockMessageEventFrom: chat_id.
 
 	self core handleEvent: message.
-	user := self class mockUserWith: (self class mockMessageID asString, chat_id).
+	user := TCTMocks mockUserWith: (TCTMocks mockMessageID asString, chat_id).
```

Comment:
Why do we mock a User with a MessageID?

Reply:
That has nothing to do with the new code. What is happening here, is that you have to give the extra attribute and that has to match what was already supplied in the initial new message event. However, those are specifics of that test (which was written by you 😛 )